packages/@okta/vuepress-site/docs/release-notes/2020-09-1/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2020.09.1
+
+| Change                                                    | Expected in Preview Orgs |
+| --------------------------------------------------------- | ------------------------ |
+| [Bugs fixed in 2020.09.1](#bugs-fixed-in-2020-09-1)       | September 10, 2020           |
+
+### Bugs fixed in 2020.09.1
+
+* When attempting to reset a user's password using the [`lifecycle/reset_password` endpoint](/docs/reference/api/users/#reset-password), admins received a 500 `Internal Server` error code rather than a valid error message if the user's email address was invalid. (OKTA-307089)
+* If a [`Groups` claim](/docs/guides/customize-tokens-returned-from-okta/-/create-groups-claim/) returned more than 100 groups, then tokens couldn't be minted, which generated a `500 Internal Server` error code instead of a `400 Bad Request` error code. (OKTA-321988)
```

Comment:
```suggestion
* If a [`Groups` claim](/docs/guides/customize-tokens-returned-from-okta/-/create-groups-claim/) returned more than 100 groups, then tokens couldn't be minted, which generated an HTTP 500 error instead of an HTTP 400 error. (OKTA-321988)
```

Reply:
Fixed and also in the third blurb.